websites/C/Coub/presence.ts

```diff
@@ -231,19 +227,17 @@ const pages: PageContext[] = [
 						: images.PAUSE;
 
 				if (showWatch) {
-					data.buttons.push(
-						...[
-							{
-								label: strings.watchVideo,
-								url: document.location.href,
-							},
-							...getSourceLink(
-								activeMedia.parentElement.querySelector<HTMLAnchorElement>(
-									'.coub__info .media-block__item > a[type="embedPopup"]'
-								)?.href
-							),
-						]
-					);
+					data.buttons = [
+						{
+							label: strings.watchVideo,
+							url: document.location.href,
+						},
+						getSourceLink(
+							activeMedia.parentElement.querySelector<HTMLAnchorElement>(
+								'.coub__info .media-block__item > a[type="embedPopup"]'
+							)?.href
+						)?.[0],
```

Comment:
getSourceLink never returns an array with more than 1 element, why is it even returning an array to begin with?